stable/minio/templates/pvc.yaml

```diff
@@ -20,8 +20,16 @@ spec:
   resources:
     requests:
       storage: {{ .Values.persistence.size | quote }}
+      
 {{- if .Values.persistence.storageClass }}
-  storageClassName: {{ .Values.persistence.storageClass | quote }}
+{{- if (eq "-" .Values.persistence.storageClass) }}
```

Comment:
is this really needed?

Reply:
@maorfr Yes, because if you use storageClass by default, it's create volume for nas with storage class (its's not correct)

Reply:
@maorfr Any examples 
https://github.com/helm/charts/blob/master/stable/stolon/templates/keeper-statefulset.yaml#L183
https://github.com/helm/charts/blob/master/stable/hoard/templates/pvc.yaml#L23
https://github.com/helm/charts/blob/master/incubator/druid/templates/historical/statefulset.yaml#L132
https://github.com/helm/charts/blob/master/incubator/druid/templates/historical/statefulset.yaml#L132
https://github.com/helm/charts/blob/master/stable/mariadb/templates/master-statefulset.yaml#L239
https://github.com/helm/charts/blob/master/stable/chartmuseum/templates/pvc.yaml#L19
https://github.com/helm/charts/blob/master/stable/influxdb/templates/pvc.yaml#L18